app/views/layouts/exercise_inputs/editors/_custom.html.erb

```diff
@@ -0,0 +1,6 @@
+<div>
+  <%= render partial: 'layouts/runner_assets', locals: {language: @exercise.language, prefix: 'custom_editor'} %>
```

Comment:
Lastly we believe that we should keep the same naming metaphor on bridge and runner metadata, something like this: 


```ruby
layout_assets_urls: {  # we are not sure about which prefix to use, another option was view_assets_url. However, it is important to have one in order avoid ambiguities 
    js: [...],
    htmls: [...],
    css: [...],
},
editor_assets_urls: { # we suggest removing the custom keyword, since all assets exposed by the runner are actual customizations 
    js: [...],
    htmls: [...],
    css: [...],
},
submit_assets_urls: { # in the near feature we will need this
    js: [...],
    htmls: [...],
    css: [...],
}
```